monai/networks/nets/vitautoenc.py

```diff
@@ -78,9 +79,14 @@ def __init__(
         """
 
         super().__init__()
-
+        if not is_sqrt(patch_size):
```

Comment:
this looks necessary as the two deconv layers will upsample by ratio of `sqrt(patch_size)` twice. cc @finalelement